modules/oe_content_persistent/tests/src/Functional/PersistentUrlControllerTest.php

```diff
@@ -38,6 +38,9 @@ protected function setUp() {
 
     $node_type = NodeType::create(['type' => 'page']);
     $node_type->save();
+
+    $user = $this->drupalCreateUser(['access content']);
```

Comment:
Why is this needed actually?

Reply:
related to https://github.com/openeuropa/oe_content/pull/98